### ISSUE TITLE
Unhandled Rejection when `.jscpd` folder is missing on macOS

### ISSUE THREAD

**Issue by lo1tuma** [kucherenko/jscpd#204]
**Describe the bug**

With the latest version of `jscpd` I get an unhandled rejection warning on macOS devices regarding a missing `.jscpd` folder.

```
(node:49306) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, scandir '.jscpd'
    at Object.readdirSync (fs.js:808:3)
    at /my-project/node_modules/jscpd/build/stores/stores-manager.js:64:35
(node:49306) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:49306) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

We are using jscpd only since a few weeks and I’m quite sure at the time we introduced it, this problem did not exist.
We noticed it today because we exceeded our threshold of allowed amount of duplications. The program exists with exit code `0`.

On our CI, which is a linux server, this problem doesn’t exist and `jscpd` exits with exit code `1` as expected.

**To Reproduce**
1. Delete the `.jscpd` folder if it exists
1. Configure `jscpd` to fail when a certain threshold is reached
1. Run `jscpd .` on project where the threshold is reached

**Expected behavior**

I would expect to not see a unhandled rejection warning and that the program exists with exit code `1`.

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version: 10.13.6
 - NodeJS Version: 11.5
 - jscpd version: 2.0.6

**Comment by kucherenko** [comment]
thank you for the report, will fix it

**Comment by kucherenko** [comment]
fixed at jscpd@2.0.7

**Comment by lo1tuma** [comment]
Awesome, thank you!
